expression/builtin_op.go

```diff
@@ -332,6 +332,7 @@ func (c *leftShiftFunctionClass) getFunction(ctx sessionctx.Context, args []Expr
 	bf := newBaseBuiltinFuncWithTp(ctx, args, types.ETInt, types.ETInt, types.ETInt)
 	sig := &builtinLeftShiftSig{bf}
 	sig.tp.Flag |= mysql.UnsignedFlag
+	sig.setPbCode(tipb.ScalarFuncSig_LeftShift)
```

Comment:
Shall we add a test case for this?

Reply:
This line is added in #13334 , so I removed it in the latest commit.